_maps/RandomRuins/SpaceRuins/spacehotel.dmm.rej

```diff
@@ -0,0 +1,9 @@
+diff a/_maps/RandomRuins/SpaceRuins/spacehotel.dmm b/_maps/RandomRuins/SpaceRuins/spacehotel.dmm	(rejected hunks)
+@@ -4525,7 +4522,6 @@
+ /area/ruin/hotel/security)
+ "ly" = (
```

Comment:
I hope you copied over their maps rather than fix the maps by hand.

Another reject